.github/workflows/ci-cd.yml

```
@@ -0,0 +1,527 @@
+
+      - name: Run smoke tests
+        run: |
+          export STAGING_URL="${{ secrets.STAGING_URL }}"
+          npm run test:smoke -- --baseUrl=$STAGING_URL
+
+      - name: Notify deployment
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          channel: '#deployments'
+          webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)